db/migrate/20220106181004_add_status_to_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToItems < ActiveRecord::Migration[5.2]
```

Comment:
didn't you add this migration already in a different PR? Might be worth double checking that we don't have two columns added.